composer.json

```diff
@@ -13,30 +13,30 @@
         }
     },
     "require": {
-        "php": "^5.5 || ^7.0",
+        "php": "^5.6 || ^7.0",
         "zendframework/zend-stdlib": "^2.7 || ^3.0"
     },
     "require-dev": {
-        "zendframework/zend-config": "^2.6",
-        "zendframework/zend-console": "^2.6",
-        "zendframework/zend-http": "^2.5.4",
-        "zendframework/zend-i18n": "^2.6",
-        "zendframework/zend-log": "^2.7.1",
-        "zendframework/zend-mvc": "^2.7.9 || ^3.0",
-        "zendframework/zend-router": "^3.0",
+        "zendframework/zend-config": "^2.6 || ^3.1",
+        "zendframework/zend-console": "^2.6.0",
```

Comment:
You are right – maybe. Is this relevant for Composer? Otherwise these requirements are nowhere documented. This affects composer.json, travis.yml and more in all components.